todo_from_file.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
Should we add a flag like `--code`, or something better if you have any idea, for this command? Couldn't find it.
So if I execute `todo --code`, it generates the todo recursively?

Reply:
Never mind! Missed the `--this-dir`. All is good!